site/content/en/docs/handbook/addons/gcp-auth.md

```diff
@@ -6,13 +6,15 @@ date: 2020-07-15
 ---
 
 
-The gcp-auth addon automatically and dynamically configures pods to use your credentials, allowing applications to access Google Cloud services as if they were running within Google Cloud.  
+The `gcp-auth` addon automatically and dynamically configures pods to use your credentials, allowing applications to access Google Cloud services as if they were running within Google Cloud.  
 
-The addon normally uses the [Google Application Default Credentials](https://google.aip.dev/auth/4110) as configured with `gcloud auth application-default login`. If you already have a json credentials file you want specify, such as to use a service account, set the GOOGLE_APPLICATION_CREDENTIALS environment variable to point to that file.
+The addon defaults to using your environment's [Application Default Credentials](https://google.aip.dev/auth/4110), which you can configure with`gcloud auth application-default login`. 
```

Comment:
```suggestion
The addon defaults to using your environment's [Application Default Credentials](https://google.aip.dev/auth/4110), which you can configure with `gcloud auth application-default login`. 
```

Reply:
Thanks for catching this — amended the commit.